Modules/Module_Processing/Module_ASL/xASL_wrp_Quantify.m

```diff
@@ -73,9 +67,19 @@ function xASL_wrp_Quantify(x, PWI4D_Path, pathOutputCBF, M0Path, SliceGradientPa
 if nargin<5 || isempty(SliceGradientPath)
     SliceGradientPath = x.P.Pop_Path_SliceGradient_extrapolated;
 end
+
 if ~isfield(x.modules.asl,'bUseBasilQuantification') || isempty(x.modules.asl.bUseBasilQuantification)
    x.modules.asl.bUseBasilQuantification = false;
 end
+
+if nargin<6 || isempty(bSaveCBF4D)
+	bSaveCBF4D = false;
+end
+
+if x.modules.asl.bUseBasilQuantification && bSaveCBF4D
```

Comment:
BASIL doesn't produce CBF4D. It does fitting to all data.
To have CBF4D, you would have to run BASIL separately for each dynamics. 

So it would be easy, but it would take enormous amount of time. Same why we don't do BASIL in standard space.